internal/api/schemas/v1.yaml

```diff
@@ -828,6 +845,46 @@ components:
           minLength: 1
         weight:
           type: number
+    HealthStatus:
+      type: object
+      required:
+      - controllers
+      - gateways
+      properties:
+        controllers:
+          type: array
+          items:
+            "$ref": "#/components/schemas/ControllerHealth"
+        gateways:
+          type: array
+          items:
+            "$ref": "#/components/schemas/GatewayHealth"
+    ControllerHealth:
+      type: object
+      required:
+      - name
+      - id
+      - status
+      properties:
+        name:
+          type: string
+        id:
+          type: string
+        status:
+          type: string
+    GatewayHealth:
+      type: object
+      required:
+      - name
+      - id
+      - status
+      properties:
+        name:
+          type: string
+        id:
+          type: string
+        status:
+          type: string
```

Comment:
Makes sense to split this for future-proofing. Just curious if you have any ideas for how they might diverge at the moment?

Reply:
not entirely sure -- I/whoever else might finish up the implementation (with the gateway querying part of this) should make sure to try and review the reqs to see if there's any information in general that we want to convey via our CLI tool. We should also probably check in with @jherschman once we have some of that baseline info displayed just to make sure everything at the command line is both useful and pretty.